src/logarithm.jl

```diff
@@ -9,6 +9,24 @@ function Base.float(x::Level{L,S}) where {L,S}
     return Level{L,S,typeof(v)}(v)
 end
 big(x::Level{L,S}) where {L,S} = Level{L,S}(big(x.val))
+"""
+    logunit(x::LogScaled)
+    logunit(x::Union{Type{<:LogScaled}, MixedUnits})
+Returns the units associated with a `LogScaled` instance, a `LogScaled` type,
```

Comment:
```suggestion
Returns the logarithmic "units" associated with a `LogScaled` instance, a `LogScaled` type,
```
The [manual](https://painterqubits.github.io/Unitful.jl/stable/logarithm/) refers to these as `logarithmic "units"`. I think this might be less confusing, especially in the case of `MixedUnits`.